Sources/ProjectSpec/Source.swift

```diff
@@ -43,6 +45,10 @@ extension Source: JSONObjectConvertible {
         let maybeCompilerFlagsArray: [String]? = jsonDictionary.json(atKeyPath: "compilerFlags")
         compilerFlags = maybeCompilerFlagsArray ??
             maybeCompilerFlagsString.map{ $0.split(separator: " ").map{ String($0) } } ?? []
+        let maybeExcludesString: String? = jsonDictionary.json(atKeyPath: "excludes")
```

Comment:
We don't need to allow a string version like the compiler flags. Compiler flags were special as they are often defined as a single string in xcode and xcconfig files